source/Government.cpp

```diff
@@ -207,6 +212,7 @@ double Government::InitialPlayerReputation() const
 double Government::PenaltyFor(int eventType) const
 {
 	double penalty = 0.;
+
```

Comment:
Another whitespace change that needs scrubbed.
```suggestion
```

Reply:
euh sure but I added that caus it felt better a bit spaced
also misread the previous suggestion thinking you wanted the line but nothing on it eh watever next time I'll just hit commit suggestion doesn't matter if there is 1 for each thing you ask
I've changed editor btw normally it should always use tabs now not spaces (caus the old one just didnt want to work sorry for loss time)